docs/users/navigation/joysticks.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Joystick Navigation
+layout: default
+
+grand_parent: Users
+parent: Navigation
+nav_order: 2
+---
+
+# Joystick Navigation
+In addition to the normal navigation in OpenSpace using the keyboard and mouse it is also possible to navigate using a game controller, such as an Xbox controller, PS4 controller or a SpaceMouse. If you want to use any of these supported controllers you need to include their asset file into a profile. This can be done in the profile editor in the launcher by editing the profile to include one of the joystick assets. All of the joystick assets are located in the sub-folder <code>data/assets/util/joysticks</code> inside the OpenSpace folder. It is important that the right asset file is included for the type of game controller you are using. For example the wireless Xbox asset will not work with a non-wireless Xbox controller and vice versa.
+
+## Xbox Controller
+The navigation using an Xbox or PS4 controller are very similar, the only difference is the layout of the controllers. Otherwise the functionality is the same. The image below shows a map over the buttons and joysticks on a Xbox controller.
+
+![](images/xbox.png)
+
+The navigation using a Xbox controller in OpenSpace is defined in the Xbox asset file <code>(xbox.asset)</code> or the Xbox wireless asset file <code>(xbox-wireless.asset)</code>. The table below will give an overview of what each button or joystick on the Xbox controller does in OpenSpace. NA in the table specifies that this button or joystick has no functionality in the default version of OpenSpace.
```

Comment:
The navigation using a Xbox controller in OpenSpace is defined in the Xbox asset file (<code>xbox.asset</code>) or the Xbox wireless asset file (<code>xbox-wireless.asset</code>). The table below gives an overview of what each button or joystick on the Xbox controller does in OpenSpace. NA in the table specifies that this button or joystick has no functionality in the default version of OpenSpace.